### PR TITLE
Add the ability to hide warnings

### PR DESCRIPTION
**Pull request by ablyler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Format the values for these config options per the [PHP_CodeSniffer documentatio
 
 1. file_extensions - This is where you can configure the file extensions for the files that you want PHP_CodeSniffer to analyze.
 2. standard - This is the list of standards that you want PHP_CodeSniffer to use while analyzing your files.
+3. ignore_warnings - You can hide warnings, and only report errors with this option.
```
Comment (GordonDiggs) [inline README.md]: I know it's not entirely your change, but could we turn this into an unordered list instead of an ordered list?

Comment (ablyler) [inline README.md]: Sure.

Comment (GordonDiggs) [inline README.md]: Thank you!

**Comment by GordonDiggs** [comment]
Looks good! Will push up a new docker image after I merge

**Pull request merged**